README.md

```diff
@@ -39,7 +39,7 @@ If your AWS credentials are in the default profile, you can omit the `AWS_PROFIL
 
 With all the credentials in place, running
 ```
-$ s3same --repo some-repo-name --owner some-user --pro
+$ s3same some-repo-name --owner some-user --pro
```

Comment:
The order does not matter.  I don't have that level of control over the usage printout (and what I put in the README is the actual printout).